app/services/risk_assessment/high_risk_assessment.rb

```diff
@@ -29,7 +29,7 @@ def enhancement_applied?
     end
 
     def uplift_applied?
-      @items[:work_items].work_item_forms.filter_map(&:uplift).any?
+      @items[:work_items].work_item_forms.any? { _1.uplift&.positive? }
```

Comment:
Just thought of a bug in this. Repro:

1. create an NSM claim and check the 'enhanced rates claimed' checkbox on the 'reason for claim' screen
2. add a work item with an uplift applied
3. From the check answers screen go back to the 'reason for claim' screen and uncheck 'enhanced rates claimed'
4. Re-fill the form without editing your original work item
5. Submit the form.

As the uplift stored in the database is a positive number, this line of code will flag the claim as high risk even though, since uplift cannot be applied as enhanced rates are not claimed, it should not be.

To get around this, we can use the `apply_uplift` method on the work item form, like so:

```ruby
 @items[:work_items].work_item_forms.any?(&:apply_uplift)
 ```

This method already contains the logic we need to prevent false positives while also dealing with uplift being nil.